src/data/codingcontracttypes.ts

```diff
@@ -1422,9 +1422,13 @@ export const codingContractTypesMetadata: ICodingContractTypeMetadata[] = [
     },
     solver: (_data: unknown, ans: string): boolean => {
       const data = _data as [number, [number, number][]];
+
+      //Sanitize player input
+      const sanitizedPlayerAns: string = removeBracketsFromArrayString(ans);
+
       //Case where the player believes there is no solution.
       //Attempt to construct one to check if this is correct.
-      if (ans == "[]") {
+      if (sanitizedPlayerAns === "") {
```

Comment:
Both `"[]"` and `""` string inputs will be accepted, and both are possible string representations of empty array.